config/eslint/.eslintrc-test

```diff
@@ -6,6 +6,12 @@ extends:
 env:
   mocha: true
 
+plugins:
+  - mocha
+
+rules:
+  mocha/no-exclusive-tests: 2
```

Comment:
Actually, it should be left as is. It is a scaler value it is not an array like the other things in this file